### ISSUE TITLE
Having trouble building on Mac

### ISSUE THREAD

**Issue by briancullinan** [iortcw/iortcw#83]
Getting this error after lots of files successfully parsed. I am new to the C toolchain.

```
REF_CC build/release-darwin-x86_64/rend2/glsl/bokeh_fp.c
clang: warning: libstdc++ is deprecated; move to libc++ with a minimum deployment target of OS X 10.9 [-Wdeprecated]
ld: library not found for -lstdc++
clang: error: linker command failed with exit code 1 (use -v to see invocation)
REF_CC build/release-darwin-x86_64/rend2/glsl/bokeh_vp.c
make[2]: *** [build/release-darwin-x86_64/iowolfmp.x86_64] Error 1
make[2]: *** Waiting for unfinished jobs....

```

I have ioquake3 building fine. I can't figure out which part of the Makefile to modify. I am running the make-macos_xcode5.sh script because make-macos-ub.sh says it only works on 10.5, I'm on 10.15, xcode Version 11.3.1 (11C504).

**Comment by MAN-AT-ARMS** [comment]
I have no way to test anything on that version of OSX / XCode.
I would start with trying an older version of XCode since it is showing a minimum depoyment target of 10.9...
That's not going to work.

**Comment by tomkidd** [comment]
No, I've fixed this and I've never done the work to do a fork, PR, etc. 

Basically the script works fine if you update the version required in the Makefile

<img width="553" alt="image" src="https://user-images.githubusercontent.com/502580/75184111-7788d600-5709-11ea-8df3-43695c3e261e.png">

You'll also need to add this bit to the part that generates the plist in the build script, which despite being named "xcode5" works just fine in Xcode 11, etc.

<img width="794" alt="image" src="https://user-images.githubusercontent.com/502580/75184339-db130380-5709-11ea-9fde-9294c8398daf.png">

I'll get on forking and doing a PR but in the meantime if you want to do a little hand-editing you can be off to the races right now

**Comment by MAN-AT-ARMS** [comment]
Unfortunately it has to be able to build on systems other than yours, so I can not merge that change

**Comment by tomkidd** [comment]
OK, I'll see if I can figure out a more version-agnostic way of accomplishing it and leave the UB version intact in the process. Probably rope in some changes from what ioquake3 is doing, since their take on the makefile looks to be sniffing out what is installed. 

At the very least I'd say multiple version-specific Xcode scripts from years back shouldn't be necessary.

**Comment by MAN-AT-ARMS** [comment]
There is a test for Tiger in the Xcode build script...

Open a terminal and see if this spits out a 1 on your system. (This has been modified to test for Catalina which is 64-bit only)


uname -r |perl -w -p -e 's/\A(\d+)\..*\Z/$1/; $_ = (($_ >= 19) ? "1" : "0");'`

**Comment by tomkidd** [comment]
Once I remove the stray caret at the end it gives me a "1"

**Comment by MAN-AT-ARMS** [comment]
I put up a test build script for XCode 10+ in the SP folder. Can you try that and see if it works?

**Comment by tomkidd** [comment]
It doesn't, both due to some syntax errors and also that XCODE_VERSION_MAJOR isn't resolving to anything. And even when I'm short circuiting the logic it's still not using 10.9 stuff for some reason.

There's definitely got to be some way to do this that keeps everything happy, just have to figure it out. I tried incorporating what ioquake3 is doing and it's not working and I'm not clear on why but my hatred of working with Makefiles is definitely solidifying.

I'll fiddle with it some more but in the meantime anyone who's seeing this and is willing to do the hack thing I showed above with the Makefile can at least build.

**Comment by briancullinan** [comment]
Thank you @tomkidd Will try this soon, working on building lots of mods for QuakeJS. Wanted to try the renderer on iortcw.
